build.rs

```diff
@@ -9,5 +10,9 @@ fn main() {
         autocfg::emit("has_i128");
     }
 
+    // The RangeBounds trait was stabilized in 1.28, so from that version onwards we
+    // implement that trait.
+    AutoCfg::new().unwrap().emit_rustc_version(1, 28);
```

Comment:
Please save the previous `autocfg::new()` in a local and then reuse that here. Otherwise, it repeats some startup checks.